CodePush.m

```diff
@@ -36,26 +36,41 @@ @implementation CodePush {
 // that is associated with an update's package.
 static NSString *const PackageHashKey = @"packageHash";
 static NSString *const PackageIsPendingKey = @"isPending";
+static NSString *const BinaryBundleDateKey = @"binaryDate";
+
+// These values are used to save the bundleURL and extension for the JS bundle
+// in the binary.
+static NSString *binaryJsName = @"main";
```

Comment:
How about `bundleResourceName` and `bundleResourceExtension`? Those match the parameter names we use for `bundleURL`.